### PR TITLE
content-repo: add resource limits/requests everywhere

### PR DESCRIPTION
**Pull request by majewsky**

### CODE REVIEW

File: openstack/content-repo/templates/check-deployment.yaml [hunk 1]
```diff
@@ -54,5 +54,13 @@ spec:
             name: secret
           - mountPath: /bin-check
             name: bin
+        resources:
+          # observed usage in qa-de-1: CPU <= 0.1m, RAM = 3-4 MiB
+          limits:
+            cpu: 'l0m'
```
Comment (reimannf) [inline openstack/content-repo/templates/check-deployment.yaml:60]: what is `l0m` btw?

Comment (majewsky) [reply openstack/content-repo/templates/check-deployment.yaml:60]: 10 milli, i.e. 1% of a core

Comment (reimannf) [reply openstack/content-repo/templates/check-deployment.yaml:60]: you want `10m` or `l0m` ?

**Review by reimannf — APPROVED** [review]

**Pull request merged**